content/docs/news/release_announcements/keptn-0170/index.md

```diff
@@ -0,0 +1,170 @@
+* **installer:** Keptn's Helm charts were reworked and some values were changed and/or moved. Please consult the [upgrade guide](https://keptn.sh/docs/install/upgrade/#upgrade-from-keptn-0-16-x-to-keptn-0-17-x) to make sure your installation can be upgraded successfully. With this change, users now have the option to customize resource limits/requests and to add custom sidecars and extra volumes from the Helm values.
+
+### Features
+
+* Adapt go-utils changes in git credentials models ([#8020](https://github.com/keptn/keptn/issues/8020)) ([e8e2e6c](https://github.com/keptn/keptn/commit/e8e2e6cfe7af4b30f4071de6aef38ecdc12907c7))
+* Add headers to git provisioner ([#8132](https://github.com/keptn/keptn/issues/8132)) ([f02aeba](https://github.com/keptn/keptn/commit/f02aeba6d384b9d7bb2aa50ce1796fc9e4cb3d6d))
+* Add OAuth scopes to swagger and add possibility to disable deprecated APIs ([#8051](https://github.com/keptn/keptn/issues/8051)) ([0dc1203](https://github.com/keptn/keptn/commit/0dc12034548d1003bd02f331979e7219f4557d73))
+* **api:** Create import endpoint ([#8137](https://github.com/keptn/keptn/issues/8137)) ([75ae009](https://github.com/keptn/keptn/commit/75ae0093c16ebb0396ee0eca3ac6c47592510a19))
+* **bridge:** Add approval-item-module ([#8069](https://github.com/keptn/keptn/issues/8069)) ([15050ba](https://github.com/keptn/keptn/commit/15050ba60ce48a171d2ccf9926c480243e623f37))
+* **bridge:** Add deletion dialog module ([#8060](https://github.com/keptn/keptn/issues/8060)) ([bac2bc8](https://github.com/keptn/keptn/commit/bac2bc8e2fd1870ccf188e504c5b02eab1d155f5))
+* **bridge:** Add ktb-confirmation-dialog module ([#8058](https://github.com/keptn/keptn/issues/8058)) ([dfc286e](https://github.com/keptn/keptn/commit/dfc286e516e2ed185335c94832f202fbddc0b752))
+* **bridge:** Add ktb-copy-to-clipboard module ([#8072](https://github.com/keptn/keptn/issues/8072)) ([473fce5](https://github.com/keptn/keptn/commit/473fce5baa601fee106af1d1c4b409e7cc74911c))
+* **bridge:** Add ktb-create-service-module ([#8073](https://github.com/keptn/keptn/issues/8073)) ([ff73348](https://github.com/keptn/keptn/commit/ff73348b5a817981195d76ade00334463fbc541a))
+* **bridge:** Add ktb-loading module ([#8048](https://github.com/keptn/keptn/issues/8048)) ([b6717fd](https://github.com/keptn/keptn/commit/b6717fdd1a7beee657a9fbccd5c609ee900974ef))
+* **bridge:** Add modules in a bulk (01) ([#8077](https://github.com/keptn/keptn/issues/8077)) ([eeef827](https://github.com/keptn/keptn/commit/eeef827260015b8aa4b0bbf669b4f3926e9f4f3f))
+* **bridge:** Add modules in a bulk (02) ([#8091](https://github.com/keptn/keptn/issues/8091)) ([1cc9a44](https://github.com/keptn/keptn/commit/1cc9a44e47f6505af3e6124d6e76f54580ab628d))
+* **bridge:** Add modules in a bulk (03) ([#8125](https://github.com/keptn/keptn/issues/8125)) ([a28be76](https://github.com/keptn/keptn/commit/a28be76bca7bc05d3af5139d9afba376d2433c5e))
```

Comment:
```suggestion
* **bridge:** Add modules in bulk (03) ([#8125](https://github.com/keptn/keptn/issues/8125)) ([a28be76](https://github.com/keptn/keptn/commit/a28be76bca7bc05d3af5139d9afba376d2433c5e))
```
Do you want to have three items with the same description or do you want a single item that lists three issues?